src/rust/uid-allocator/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "uid-allocator"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+name = "uid_allocator"
+
+[[bin]]
+name = "uid-allocator"
+path = "src/main.rs"
+
```

Comment:
note: jgrillo recently pointed out that if you have a src/lib.rs and src/main.rs you don't need to explicitly add this [lib] or this [[bin]].

https://stackoverflow.com/questions/57756927/rust-modules-confusion-when-there-is-main-rs-and-lib-rs/57767413#57767413

Reply:
til